seedfarmer/mgmt/metadata_support.py

```diff
@@ -164,10 +165,10 @@ def convert_cdkexports(
         data = cdk_output[out_key]["metadata"]
     else:
         clean_jq_path = _clean_jq(jq_path)
-        jq_command = f"cat {cdk_output_path} | jq '{clean_jq_path}'"
         _logger.info("Pulling with jq path '%s' from %s file", clean_jq_path, json_file)
-        _logger.debug(f"The entire jq command: {jq_command}")
-        os.system(f"{jq_command} > tmp-metadata")
+        with open("tmp-metadata", "w") as outfile:
+            cat_process = subprocess.Popen(["cat", cdk_output_path], stdout=subprocess.PIPE, shell=False)
+            subprocess.run(["jq", clean_jq_path], stdin=cat_process.stdout, stdout=outfile, shell=False)
```

Comment:
I think this is rebuilding what you can do with subprocess directly? Does this work:

```suggestion
        with open(cdk_output_path, "r") as infile, open("tmp-metadata", "w") as outfile:
            subprocess.run(["jq", clean_jq_path], stdin=infile, stdout=outfile, shell=False)
```

Reply:
Oh yeah that's better. Thanks!